index.html

```diff
@@ -17,22 +17,37 @@
                     <li>
                     <a href="#"> <img src="images/logo.svg" alt="logo"> </a>
                     </li>
+                     
+                    <li >
+
+                <ul class="nav-item"> 
+                    <li >
+                        <a href="#"> Portolio </a>
+                        </li> 
                     <li>
+                    <a href="#"> About</a>
+                    </li>
+                    <li >
+                    <a href="#"> Contact </a>
+                     </li>
+                    </ul>
+                </li> 
+                    <li class="menu">
                     <a href="#"> <img src="images/menu.svg" alt="menu"> </a>
                     </li>
-
+                
                 </ul>
             </nav>
 
         </header>
 
-        <section>
+        <section class="hero">
```

Comment:
- [ ] I think it's better to use the whole width of the page to present your content. Your page is presenting a blank right side when in the desktop version, creating a nonfriendly UX.

<img width="783" alt="Screen Shot 2022-11-05 at 10 49 09 PM" src="https://user-images.githubusercontent.com/47673311/200150555-beb02623-be6e-4ac0-8842-587acc3870ab.png">